enterprise/backend/src/metabase_enterprise/serialization/serialize.clj

```diff
@@ -238,3 +238,18 @@
 (defmethod serialize-one NativeQuerySnippet
   [snippet]
   (select-keys snippet [:name :description :content]))
+
+(defmethod serialize-one Metric
```

Comment:
This is v1 serdes and we really don't want to be changing it.

If v1's tests are failing because of the changes to the YAML ordering, we should just fork `metabase-enterprise.serialization.dump` into separate namespaces like everything else.

Reply:
I've dropped the commit that changed this. I've just noticed that there are tests there which do not run at all, so I've made them run, but yeah, not really necessary for the change I'm making.